### PR TITLE
Upgraded to Castle.Core 4.2.0 to 4.3.1. 

### PR DESCRIPTION
**Pull request by fir3pho3nixx**
https://github.com/castleproject/Core/releases/tag/v4.3.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 Enhancements:
+- Upgraded to Castle.Core 4.2.0 to 4.3.1. https://github.com/castleproject/Core/releases/tag/v4.3.1 (@fir3pho3nixx).
```
[the remark this answers is not in the document]
Comment (stakx) [reply CHANGELOG.md]: Small nit, I guess it should read `#413` (this PR) instead of `#409` (a previous PR).

**Comment by fir3pho3nixx** [comment]
Thanks for checking, did this at 02:00 in the morning. :)

**Comment by jonorossi** [comment]
Did I miss something? The changelog still has 2 entries for the same release:

```
Enhancements:
- Upgraded to Castle.Core 4.2.0 to 4.3.1 (@fir3pho3nixx, #413)
...
- Upgraded Castle.Core from 4.2.0 to 4.3.0 (@fir3pho3nixx, #409)
```

**Comment by fir3pho3nixx** [comment]
Will fix.

**Pull request merged**